src/screens/choice-exercises/components/SingleChoiceListItem.tsx

```diff
@@ -45,10 +43,8 @@ const StyledArticle = styled.Text`
   text-align: center;
   margin-right: 3%;
   margin-left: 3.5%;
-  color:  ${(props: { pressed: boolean; selected: boolean; correct: boolean }) => {
```

Comment:
Have your removed all of this "pressed" stuff? We need this, for the blue color, when an element is pressed, but not release. We have the same behaviour when clicking a Profession, Subprofession or Exercise and it should be here also.